ngraph/python/tests/conftest.py

```diff
@@ -25,11 +25,17 @@ def pytest_addoption(parser):
         choices=["CPU", "GPU", "FPGA", "HDDL", "MYRIAD", "HETERO"],
         help="Select target device",
     )
+    parser.addoption(
```

Comment:
Not needed.

Reply:
Needed with configurable path to additional models path.